src/shared/pages/team/manageNominatedContacts/__tests__/__snapshots__/addNominatedContact.spec.tsx.snap

```diff
@@ -0,0 +1,46 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Ideally you'd tell Git this was a moved file with `git mv`, but it's not too important.

Reply:
Thanks. I've never used git mv. I'll look into it for my next PR.